src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -541,7 +543,8 @@ public void run() {
                         frame.output(Localization.lang("Attempting SQL export..."));
                         final DBExporterAndImporterFactory factory = new DBExporterAndImporterFactory();
                         final DBExporter exporter = factory.getExporter(dbs.getServerType());
-                        exporter.exportDatabaseToDBMS(database, bibDatabaseContext.getMetaData(), null, dbs, frame);
```

Comment:
Should really only the selected entries be exported or all (getDatabase().getEntries)? Previously `null` indicated the latter.